unstable/wlr-screencontent-unstable-v1.xml

```diff
@@ -28,14 +28,14 @@
     An interface to capture surfaces in an efficient way.
     Overall usage:
 
-    1.) client registers with zwlr_screencontent_manager
+    1.) client registers with wlr_screencontent_manager
     2.) server sends client info about surfaces via "receive_surface_info"
     3.) client subscribes to capture a surface via the "capture" request
-    4.) server sends client events via the "zwlr_screencontent_frame" interface
-    5.) client finishes and informs server via the "destroy" event
+    4.) server sends client events via the "wlr_screencontent_frame" interface
+    5.) client finishes and informs server via the "frame_destroy" event
   </description>
 
-  <interface name="zwlr_screencontent_frame" version="1">
+  <interface name="wlr_screencontent_frame_v1" version="1">
```

Comment:
You need to add a "z" prefix to mark the protocol as unstable (the protocol name doesn't have it because "unstable" is explicitly included).